client/app/cloud/project/compute/loadbalancer/configure/compute-loadbalancer-configure.controller.js

```diff
@@ -0,0 +1,260 @@
+    };
+
+    self.toggle = {
+        updatedMessage : false,
+    };
+
+
+    function init () {
+
+        // Get loadbalancer pending tasks and define poller
+        self.tasks = ControllerHelper.request.getArrayLoader({
+            loaderFunction: () => IpLoadBalancerTaskService.getTasks(loadbalancerId).then(tasks => _.filter(tasks, task => _.includes(["todo","doing"], task.status))),
+            successHandler: () => startTaskPolling()
+        });
+        self.tasks.load();
+
+
+        var validatePromise;
+        // Terminate validation if params exists
+        if ($stateParams.validate) {
+            self.loaders.loadbalancer = true
+            validatePromise = OvhApiCloudProjectIplb.Lexi().validate({ serviceName : serviceName, id : $stateParams.validate }, {}).$promise
+            .then(() => {
+                $location.search("validate",null);
+                self.toggle.updatedMessage = true;
+            })
+            .catch(err => Toaster.error( [$translate.instant('cpc_loadbalancer_error'), err.data && err.data.message || ''].join(' ')))
+            .finally(() => self.loaders.loadbalancer = false);
+            $stateParams.validate = "";
+        }
+        else {
+            validatePromise = Promise.resolve("");
+        }
+        // After validation, load the loadbalancer
+        validatePromise.then(() => getLoadbalancer(true));
+
+        $scope.$on("$destroy", () => stopTaskPolling());
+        initGuides();
+    }
+
+    function initGuides() {
+        self.guides = {
+            horizon: ovhDocUrl.getDocUrl("g1773.creer_un_acces_a_horizon")
+        };
+    }
+
+    // Get servers of the default default farm of the frontend
+    function getAttachedServers() {
+        if (!self.loadbalancer.farm) {
+            return Promise.resolve([]);
+        }
+        return OvhApiIpLoadBalancing.Farm().Http().Server().Lexi().query({
+             serviceName : loadbalancerId,
+             farmId : self.loadbalancer.farm.farmId,
+        }).$promise
+        .then(serverIds =>
+            $q.all(
+                 _.map(serverIds, serverId => OvhApiIpLoadBalancing.Farm().Http().Server().Lexi().get({
+                         serviceName : loadbalancerId,
+                         farmId : self.loadbalancer.farm.farmId,
+                         serverId : serverId,
+                     }).$promise
+                 )
+            )
+        );
+    }
+
+    // Get cloud servers to add in the loadbalancer
+    function getServers() {
+        if (self.loaders.table.server) {
+            return;
+        }
+        self.loaders.table.server = true;
+        return $q.all({
+            cloudServers : OvhApiCloudProject.Instance().Lexi().query({ serviceName : serviceName }).$promise,
+            attachedServers : getAttachedServers(),
+        }).then(({cloudServers, attachedServers}) => {
+            self.attachedServers = {};
+            _.forEach(attachedServers, (attachedServer) => {
```

Comment:
FIRST: If it's an Array you could do it with ES6 for (let attachedServer of attachedServers)
BUT what you should try to do is attachedServers.map(attachedServer => {})